test-net.json

```diff
@@ -1,3 +1,61 @@
-[
-
-]
+var tokens = {
```

Comment:
This is a JSON file. it can't contain the variable names. Please check the required structure in the README file.

Reply:
ok, but will have to check how will we access these tokens. currently we access these by `window.tokens`
`tokens` is var. I will check if there is another way

Reply:
you will need to assign loaded JSON to a variable. You can use `fetch` for that:
```
fetch("https://path_to_list.json")
.then(response => {
   tokens = response.json();
})
```

Reply:
ok